core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.module.ts

```diff
@@ -75,6 +84,11 @@ class ContributorDashboardPageModule {
 
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { downgradeModule } from '@angular/upgrade/static';
+import { OpportunitiesListItemComponent } from './opportunities-list-item/opportunities-list-item.component';
+import { OpportunitiesListComponent } from './opportunities-list/opportunities-list.component';
+import { TranslationOpportunitiesComponent } from './translation-opportunities/translation-opportunities.component';
+import { TranslationModalContent } from './modal-templates/translation-modal.component';
+import { NgbModalModule } from '@ng-bootstrap/ng-bootstrap';
```

Comment:
Why do we have these imports here but not on the top of the files?

Reply:
Done